src/System.Diagnostics.TraceSource/tests/TestTraceListener.cs

```diff
@@ -119,5 +119,10 @@ public override void WriteLine(string message)
         {
             Call(Method.WriteLine);
         }
+
+        public override void Close()
+        {
+            Dispose(); // In desktop Trace.Close() calls listener.Close() on every listener and in Core we call listener.Dispose() so the tests would fail in desktop without this override. 
```

Comment:
Sounds something reasonable. Ok, so I'll update it to match what desktop does and update the tests to actually count the number of calls to Close as well. Thanks.